### DISCUSSION TITLE
Rename 'Compose structs together for better borrowing' to snappier name

### DISCUSSION THREAD

**Discussion by ZerothLaw** [rust-unofficial/patterns#238] [From Issues]
A suggested snappier name for [compose-structs](https://github.com/rust-unofficial/patterns/blob/master/patterns/compose-structs.md): 
Borrow Decomposition. 

In essence, you're decomposing what gets borrowed by the borrow checker (at least until it gets smarter with Chalk).

**Comment by programmerjake** [comment]
what about "struct partitioning"?
